SGoopas/Assets/Scripts/MasterStateMachine.cs

```diff
@@ -83,32 +90,39 @@ private IEnumerator UpdateLoadingWithProgress(AsyncOperation operation)
         float progress;
         while (!operation.isDone)
         {
-            progress = Mathf.Clamp01(operation.progress / .9f);
```

Comment:
you can remove these 3 lines, just make sure you get rid of slider / progressTxt in MasterMonoBehaviour.cs and the Master Scene